lib/lib-javadoc/src/main/resources/SQLSpecification2.csv

```diff
@@ -15,6 +15,7 @@ Aggregation Functions,Returns the sample standard deviation of expr (the square
 Aggregation Functions,"Returns the population standard variance of expr. It considers rows as the whole population, not as a sample, so it has the number of rows as the denominator. You can also use VARIANCE(), which is equivalent.",select VAR_POP(expr) from syn123
 Aggregation Functions,"Returns the sample variance of expr. That is, the denominator is the number of rows minus one. ",select VAR_POP(expr) from syn123
 Aggregation Functions,"For each group of the column 'foo', concatenate the distinct values from column 'bar' order by 'bar' descending with a separator character of '#'.  The distinct keyword is optional.  The 'order by' clause is optional.  The separator is optional, with a default separator of ','.","select foo, group_concat(distinct bar order by bar desc separator '#') from syn123 group by foo"
+User Function,"Returns the Synapse userId of the current user.","select * from syn123 where userId = CURRENT_USER()"

```

Comment:
Probably better to add at the end a category that is more generic (e.g. Other Synapse Functions)